app/src/main/java/com/example/easysurvey/dto/Survey.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.easysurvey.dto
+//may have to fix the import
+import com.google.gson.annotations.SerializedName
+
+//change the names if you guys don't like it.. also @serialized allows to get lower case information
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.